subtrack.MAUI/Services/SubscriptionsCalculator.cs

```diff
@@ -10,4 +10,13 @@ public static decimal GetMonthlyCost(this IEnumerable<Subscription> subscription
         var price = subscriptions.Sum(x => x.Cost);
         return price;
     }
+
+    public static int GetDueDays(this Subscription subscription)
+    {
+        if (subscription is null) return 0;
+        var duration = DateTime.Now.Subtract(subscription.LastPayment);
+        if (duration.Days < 0) return 0;
```

Comment:
Good check but we can probably remove this if statement since we do want to be able to see if a subscription is overdue later on and how many days has passed since